src/Features/CSharp/Portable/GenerateConstructor/CSharpGenerateConstructorService.cs

```diff
@@ -60,35 +51,6 @@ protected override bool IsClassDeclarationGeneration(SemanticDocument document,
             return false;
         }
 
-        protected override bool TryInitializeClassDeclarationGenerationState(
```

Comment:
Not needed anymore GenerateConstructor is responsible for the "new MyFoo(x, y, z)" cases, not the "class D : BaseType" cases.  Those are handled elsewhere.